docs/audit-log-filter-new.md

```diff
@@ -0,0 +1,259 @@
+# Audit Log Filter format - XML (new style)
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The filter writes the audit log filter file in XML. The XML file uses 
+UTF-8.
+
+The <AUDIT> is the root element and this element contains 
+<AUDIT_RECORD> elements. Each <AUDIT_RECORD> element contains specific 
+information about an event that is audited. 
+
+For each new file, the Audit Log Filter plugin writes the XML 
+declaration and the root element tag. The plugin writes the closing 
+</AUDIT> root element when closing the file. If the file is open, this 
+closing element is not available.
+
+```xml
+
+<?xml version="1.0" encoding="utf-8"?>
+<AUDIT>
+    <AUDIT_RECORD>
+        <NAME>Audit</NAME>
+        <RECORD_ID>0_2023-03-29T11:11:43</RECORD_ID>
+        <TIMESTAMP>2023-03-29T11:11:43</TIMESTAMP>
+        <SERVER_ID>1</SERVER_ID>
+    </AUDIT_RECORD>
+    <AUDIT_RECORD>
+        <NAME>Command Start</NAME>
+        <RECORD_ID>5_2023-03-29T11:11:45</RECORD_ID>
```

Comment:
Please update RECORD_ID elements in a similar way as for JSON format. Here first digit should be incremented by 1 for each next event. ID for the first event starts with 0.